marathon-bigip-ctlr.py

```diff
@@ -552,7 +627,85 @@ def get_apps(apps, health_check):
     return apps_list
 
 
-def create_config_marathon(bigip, apps):
+def iapp_build_definition(config, members):
+    """Create a dict that defines the 'variables' and 'tables' for an iApp.
+
+    Args:
+        config: BIG-IP config dict
+    """
+    # Build variable list
+    variables = []
+    for key in config['variables']:
+        var = {'name': key, 'value': config['variables'][key]}
+        variables.append(var)
+
+    # The schema says only one of poolMemberTable or tableName is
+    # valid, so if the user set both it should have already been rejected.
+    # But if not, prefer the new poolMemberTable over tableName.
+    tables = []
+    if 'poolMemberTable' in config:
```

Comment:
Right, if there is no poolMemberTable but there is a tableName then we configure using a "default" poolMemberTable config/schema (the elif below).  This looks like what I'd expect.